public/js/reports/c3Charts.service.js

```diff
@@ -134,6 +138,14 @@ angular.module('app.reports').service('c3Charts', function (notify, gettextCatal
             tooltip: {
                 order: function (t1, t2) {
                     return t1.id > t2.id;
+                },
+                format: {
+                    value: function (value, ratio, id, index) {
+                        if (dataFormatted[id]) {
+                            value = numeral(value).format(dataFormatted[id]);
+                        }
+                        return value;
```

Comment:
This does not seem to work for pie/donut graph. The `id` parameter contain some value instead of the column id. Can you take a look ?

Reply:
Yes, on donut chart type id corresponding to the label. So I also changed this behavior and apply only one format (only one is needed obviously for this type of chart)